packages/zowe-explorer/src/abstract/ZoweTreeProvider.ts

```diff
@@ -260,4 +260,16 @@ export class ZoweTreeProvider {
         this.mHistory.removeSession(revisedLabel);
         this.refresh();
     }
+
+    /**
+     * Expand a node
+     * @param element the node being flipped
+     * @param provider the tree view provider
+     */
+    public async expandSession(element: IZoweTreeNode, provider: IZoweTree<IZoweNodeType>) {
+        await provider.getTreeView().reveal(element, { expand: false });
```

Comment:
I did some quick testing and noticed that this fix will still work without the reveal call using `expand: false` (line 270).

Reply:
With that line commented out, there are expansion issues in the USS view:
![expansion1](https://user-images.githubusercontent.com/122040687/216377316-cadc5cab-43c9-4b65-a5cb-78661a4d963c.gif)